src/app/api/routes/alerts.py

```diff
@@ -93,11 +94,20 @@ async def get_alert(alert_id: int = Path(..., gt=0), _=Security(get_current_acce
 
 
 @router.get("/", response_model=List[AlertOut], summary="Get the list of all alerts")
-async def fetch_alerts(_=Security(get_current_access, scopes=[AccessType.admin])):
+async def fetch_alerts(requester=Security(get_current_access, scopes=[AccessType.admin, AccessType.user]),
+                       session=Depends(get_session)):
```

Comment:
If it doesn't fit on one line, let's switch to black formatting instead for consistency :ok_hand:

Reply:
You mean applying black?
So get rid of flake8?

Reply:
I mean, if you split the signature into multiple lines, then let's use one line per argument :)

Reply:
Ok done